contentlayer.config.ts

```diff
@@ -59,7 +59,17 @@ const computedFields: ComputedFields = {
       const regXHeader = /\n(?<flag>#{1,3})\s+(?<content>.+)/g
       const slugger = new GithubSlugger()
 
-      const headings = Array.from(doc.body.raw.matchAll(regXHeader)).map(({ groups }) => {
+      // Remove the codeblocks from the doc before generating ToC Headings
+      const bodyWithoutCodeblocks = doc.body.raw.split('```').reduce((acc, ele, index) => {
+        // Every alternate block of text would be outside of the codeblock
+        if (index %2 === 0)  {
+          acc += ele;
+        }
+
+        return acc;
+      }, '');
+
+      const headings = Array.from(bodyWithoutCodeblocks.matchAll(regXHeader)).map(({ groups }) => {
```

Comment:
We can write this to be clear.  We should use regex.

Reply:
```
      const regXCodeBlock = /```[\s\S]*?```/g;
      // Remove code blocks from the raw content
      const contentWithoutCodeBlocks = doc.body.raw.replace(regXCodeBlock, '');
```

Reply:
Added the above changes.